css/at-rules/view-transition.json

```
@@ -23,7 +23,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": false
+              "version_added": "preview"
```

I'm wondering if we should link to [the meta bug](https://webkit.org/b/263602) to facilitate the follow-up?

```suggestion
              "version_added": "preview",
              "impl_url": "https://webkit.org/b/263602"
```